src/main/java/com/hydratereminder/command/reset/ResetCommandHandler.java

```diff
@@ -1,15 +1,37 @@
 package com.hydratereminder.command.reset;
 
-import javax.inject.Singleton;
-
+import com.hydratereminder.HydrateReminderPlugin;
+import com.hydratereminder.chat.ChatMessageSender;
 import com.hydratereminder.command.CommandHandler;
+import java.time.Instant;
+import java.util.Optional;
+import javax.inject.Singleton;
 
 @Singleton
 public class ResetCommandHandler implements CommandHandler {
 
+    private final HydrateReminderPlugin hydrateReminderPlugin;
+
+    private final ChatMessageSender chatMessageSender;
+
+    public ResetCommandHandler(HydrateReminderPlugin hydrateReminderPlugin, ChatMessageSender chatMessageSender) {
```

Comment:
Will it work without @inject on the constructor level? Did you try to run the game with that change?

Reply:
Nope without `@Inject` it's not working.

Reply:
So you should add it and fix it ;)

Reply:
I added it previously but somehow it got missed in the commit thanks for pointing it, I have added that now.